website/content/en/preview/tasks/pod-density.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control Pod Density"
+linkTitle: "Control Pod Density"
+weight: 20
+---
+
+Pod density is the number of pods per node. 
+
+Kubernetes has a default limit of 110 pods per node. Karpener shares this default. 
+
+## Max Pods
+
+Do not use the `max-pods` argument to kubelet. Karpenter is not aware of this value. For example, Karpenter may provision an instance expecting it to accommodate more pods than this static limit. 
```

Comment:
I feel this paragraph might confuse people. We don't provide a way, through Karpenter, to even specify max-pods. So, this would only matter to those using a custom launch template/AMI. I get what you're trying to say, but I think it might only cause more confusion.

Reply:
How common is setting a max pods value? If it's common I want to document that we don't support it. I understand it may cause confusion, so I may move this deeper in the page.